### DISCUSSION TITLE
Dropdowns in "Merge models" tab "issue"

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#1173] [General]
I don't know if it's considered as issue so I post here first, but ... I just created a custom inpainting model from a checkpoint and this requires all three dropdowns populated. When it finished, I noticed, there is no way to clear the dropdowns once it was populated with an entry.
The problem now arises, when I now want to merge two models. It's not possible because I can't clear the third dropdown. The only way would be a reload of the webpage, but that's ... just not right. 🙄

Anything that can be done about that?

**Comment by vladmandic** [comment]
Doable & makes sense - can you open a feature request so I can track it?

**Reply by Woisek** [reply]
Thanks, I'm on it.
